samples/LoggingTracer/LoggingTracer.AspNetCore/Pages/Error.cshtml

```diff
@@ -0,0 +1,26 @@
+﻿@page
```

Comment:
Did you simply took a default template? Unless you need it - I'd suggest to start with the empty apps. Otherwise we will need to update examples and code that we do not even use with the releases of ASP.NET Core.

Reply:
I changed this to the "emtpy" template. It's now a completely stripped down example.